.github/actions/dockerScoutScan/action.yml

```
@@ -39,10 +39,10 @@ runs:
         password: ${{ inputs.token }}
 
     - name: Get latest tag
-      if: ${{ inputs.compare-to-regex }}
+      if: ${{ inputs.compare-to-suffix }}
       id: get-latest
       shell: bash
-      run: echo "tag=$(curl -s 'https://registry.hub.docker.com/v2/repositories/solarwinds/solarwinds-otel-collector/tags/' | jq -r '.results[].name' | grep -E '${{ inputs.compare-to-regex }}' | sort -r | head -n 1)" >> $GITHUB_OUTPUT
+      run: echo "tag=$(grep -oP '(?<=const Version = ")[^"]+' "./pkg/version/version.go")${{ inputs.compare-to-suffix }}" >> $GITHUB_OUTPUT
```

```suggestion
      run: echo "tag=$(grep -oP '(?<=const Version = ")[^"]+' "./pkg/version/version.go")-${{ inputs.compare-to-suffix }}" >> $GITHUB_OUTPUT
```

NIT: added `-` and then `compare-to-suffix: nanoserver-xxxx`